Dockerfile

```diff
@@ -1,14 +1,20 @@
-FROM tiangolo/uwsgi-nginx-flask:python3.6-alpine3.7
+FROM tiangolo/uwsgi-nginx-flask:python3.8-alpine
 
-ENV STATIC_URL /static
-ENV STATIC_PATH /var/www/app/static
+COPY . /app
 
-COPY ./requirements.txt /var/www/requirements.txt
+COPY docker/.aws /root/.aws
+COPY docker/sqlitebackup.sh /usr/local/bin/
+COPY docker/supervisor.d/ /etc/supervisor.d/
+COPY docker/prestart.sh /app/.
 
-COPY . /app
-RUN pip install -r /var/www/requirements.txt
+RUN chmod 755 /usr/local/bin/sq*
```

Comment:
[Is this setting permissions for more than `sqlitebackup.sh`?](https://github.com/brentvollebregt/hit-counter/pull/13#discussion_r453514855)